indico/modules/events/abstracts/templates/forms/rating_widget.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    <div id="{{ field.id }}" class="rating-widget">
+        <span class="rating-value">{{ '-' if field.object_data is none else field.data }}</span>
+        {% for bullet in range(cfa.rating_range[1], cfa.rating_range[0] - 1, -1) %}
+            {% set radio_id = field.id + "-" + bullet|string %}
+            <input type="radio" id="{{ radio_id }}" name="{{ field.name }}" value="{{ bullet }}"
+                   {{ 'checked' if field.data == bullet|string else ''}}>
+            <label class="bullet" title="{{ bullet }}" for="{{ radio_id }}"></label>
```

Comment:
Not sure if our CSS handles it correctly, but you could probably just wrap the input inside the label instead of giving the radio buttons IDs